### ISSUE TITLE
About  seed

### ISSUE THREAD

**Issue by alex1243423** [DAMO-NLP-SG/VCD#30]
Hello,

I have observed that the randomness in your code is being modified. Specifically, the rand_like function seems to be altering the random state. Please see the code below for reference.

```python
import torch
from transformers import set_seed
set_seed(42)
random_tensor = torch.randn_like(torch.zeros(5))
another_random_tensor = torch.randn_like(torch.zeros(5))
print(random_tensor)
print(another_random_tensor)
```
